src/lib/viewers/media/DashViewer.js

```diff
@@ -594,15 +606,52 @@ class DashViewer extends VideoBaseViewer {
      */
     loadSubtitles() {
         // Load subtitles from video, if available
-        this.textTracks = this.player.getTextTracks().sort((track1, track2) => track1.id - track2.id);
+        this.textTracks = this.player
```

Comment:
This will affect the existing controls/player, as well. Is this safe?

Reply:
It should be safe because it's an additive change, but I can also move it to L616 to be safer